### PR TITLE
feat(system_monitor): add some smart information to diagnostics

### PR DESCRIPTION
**Pull request by kk-inoue-esol**
## Description(required)

add some S.M.A.R.T information for monitoring HDD life to diagnostics

## Related PR(optional)

https://github.com/tier4/autoware.universe/pull/25

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [commit-guidelines][commit-guidelines]
- [x] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: system/system_monitor/include/hdd_reader/hdd_reader.hpp [hunk 1]
```diff
@@ -38,6 +68,9 @@ struct HDDInfo
   uint8_t temp_;        //!< @brief temperature(DegC)
   // Lowest byte of the raw value contains the exact temperature value (Celsius degrees)
   // in S.M.A.R.T. information.
+  uint64_t power_on_hours_;           //!< @brief power on hours count
+  uint64_t total_data_written_;       //!< @brief total data written
```
Comment (yukkysaito) [inline system/system_monitor/include/hdd_reader/hdd_reader.hpp:72]: What are the units? Times? MB?

**Comment by yukkysaito** [comment]
If @ito-san can confirm to work this, I can approve.

File: system/system_monitor/include/hdd_reader/hdd_reader.hpp (hunk 1 above)
Comment (kk-inoue-esol) [reply system/system_monitor/include/hdd_reader/hdd_reader.hpp:72]: The units are size, but they differ depending on devices.
In ATA : LBA block size
In NVMe : 512byte * 1,000 (and rounded)
https://nvmexpress.org/wp-content/uploads/NVM-Express-Base-Specification-2.0b-2021.12.18-Ratified.pdf
Figure 207.Data Units Written

**Comment by ito-san** [comment]
> If @ito-san can confirm to work this, I can approve.

@yukkysaito @kk-inoue-esol Confirmed in several environments.

**Review by ito-san — APPROVED** [review]

**Pull request merged**